app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -398,4 +399,23 @@ def save_node_selector_parts_inventory(entity, hashes, target = nil)
     save_inventory_multi(:node_selector_parts, entity, hashes, deletes, [:section, :name])
     store_ids_for_new_records(entity.node_selector_parts, hashes, [:section, :name])
   end
+
+  def save_container_builds_inventory(ems, hashes, target = nil)
+    return if hashes.nil?
+    target = ems if target.nil?
+
+    ems.container_builds(true)
+    deletes = if target.kind_of?(ExtManagementSystem)
+                ems.container_builds.dup
+              else
```

Comment:
This is a good spot for ternary expression:
deletes = target.kind_of?(ExtManagementSystem) ? ems.container_builds.dup : []
makes it a lot clearer and more readable.